src/components/LandingPage.js

```diff
@@ -2,7 +2,14 @@
 import { jsx } from 'theme-ui';
 
 const LandingPage = () => {
-  return <h1>This is a landing page if you are not logged in</h1>;
+  return (
+    <div>
+      <h1>Welcome to</h1>
+      <img src="OrangeLogo.png" alt="orange squares icon" />
+      <button>Sign Up!</button>
```

Comment:
You can import the `Button` component from theme-ui to get the default styling (plus any styling we have in our `theme` file)
```
import { Button } from "@theme-ui/components";
```